packages/edit-site/src/components/editor/index.js

```diff
@@ -28,11 +27,8 @@ import { store as preferencesStore } from '@wordpress/preferences';
 import WelcomeGuide from '../welcome-guide';
 import { store as editSiteStore } from '../../store';
 import { GlobalStylesRenderer } from '../global-styles-renderer';
-import useTitle from '../routes/use-title';
```

Comment:
Could we remove this hook in favor of useEditorTitle?

Reply:
not really, useEditorTitle is using useTitle internally. useTitle is more generic.